src/Amadevus.RecordGenerator.Generators/RecordPartialGenerator.cs

```diff
@@ -76,69 +55,69 @@ StatementSyntax CreateValidateInvocation()
                 return
                     ExpressionStatement(
                         InvocationExpression(
-                            IdentifierName(Names.Validate))
-                        .AddArgumentListArguments(
-                            Descriptor.Entries.Select(CreateValidateArgument).ToArray()));
+                                IdentifierName(Names.Validate))
+                            .AddArgumentListArguments(
+                                descriptor.Entries.Select(CreateValidateArgument).ToArray()));
             }
             ArgumentSyntax CreateValidateArgument(RecordDescriptor.Entry entry)
             {
                 return
                     Argument(
-                        IdentifierName(entry.IdentifierInCamelCase))
-                    .WithRefKindKeyword(Token(SyntaxKind.RefKeyword));
+                            IdentifierName(entry.IdentifierInCamelCase))
+                        .WithRefKindKeyword(Token(SyntaxKind.RefKeyword));
             }
         }
 
-        private IEnumerable<MethodDeclarationSyntax> GenerateUpdateMethod()
+        private static MethodDeclarationSyntax GenerateUpdateMethod(RecordDescriptor descriptor)
         {
-            var arguments = Descriptor.Entries.Select(x =>
+            var arguments = descriptor.Entries.Select(x =>
             {
                 return Argument(
                     IdentifierName(x.IdentifierInCamelCase));
             });
-            yield return
-                MethodDeclaration(Descriptor.Type, Names.Update)
-                .AddModifiers(SyntaxKind.PublicKeyword)
-                .WithParameters(
-                    Descriptor.Entries.Select(CreateParameter))
-                .WithBodyStatements(
-                    ReturnStatement(
-                        ObjectCreationExpression(
-                            Descriptor.Type)
-                        .WithArgumentList(
-                            ArgumentList(
-                                SeparatedList(arguments)))));
+            return
+                MethodDeclaration(descriptor.Type, Names.Update)
+                    .AddModifiers(SyntaxKind.PublicKeyword)
```

Comment:
What formatter do you use?

I preferred the previous layout where method call's dot was on the same level as the call target.

Reply:
I am the (human) formatter. 😆 

Sorry I missed on your styling. Do you want me to revert?

Reply:
I think so. Also for formatting in general, whenever I break lines within argument list, I give every argument on new line, 1 indent more than the call. I've seen a few places where arguments are both split across lines and the first is on the same as the call.

Reply:
I don't know if I got this right fcf1db2ac4cc1ab3afdeec869a424fd7e386e72e but feel free to re-format by adding to this PR.

Reply:
I still see a lot of whitespace changes in the diff of this file.

Reply:
Not sure I see your point. Can you expand?

Reply:
Whitespace I mean:

![image](https://user-images.githubusercontent.com/2671396/62361184-c9260500-b51a-11e9-9de6-93e6cf832acf.png)

Reply:
Gotchya! I think this was some re-formatting done by the IDE on paste. Hopefully that noise is gone with 79db7367eff8c08ac9f296eb775b22132c512337.